assembly-sdk-war/src/main/resources/predefined-recipes.json

```diff
@@ -1,5 +1,6 @@
 [
   {
+    "id": "predefined-84139652330",
```

Comment:
You should  definitely change this ids to something readable.
When you got 10 recipes what id should be used for 11 ? 
Use something like `predefined-jdk8-tomcat-maven`, it will not repeat because it doesn't make any sense to have several predefined recipes which the same confs.